### PR TITLE
Nc news 9

### PR DESCRIPTION
**Pull request by ismaelpaul**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -239,6 +239,46 @@ describe('/api/articles/:article_id', () => {
...
+					const allComments = response.body.comments;
+					expect(Array.isArray(allComments)).toBe(true);
+					expect(allComments.length > 0).toBe(true);
+					allComments.forEach((comment) => {
+						expect(comment).toHaveProperty('body', expect.any(String));
+						expect(comment).toHaveProperty('votes', expect.any(Number));
+						expect(comment).toHaveProperty('author', expect.any(String));
+						expect(comment).toHaveProperty('comment_id', expect.any(Number));
+						expect(comment).toHaveProperty('created_at');
+					});
```
Comment (Rayhaan2k9) [inline __tests__/app.test.js:262]: We do need another test here. What if an article exists but doesn't have any comments? We want this to be a 200 and send the empty array

File: models/articles.models.js [hunk 2]
```diff
@@ -73,3 +79,30 @@ exports.updateArticleVotes = (incrementVoteBy, articleId) => {
...
+	WHERE articles.article_id = $1;
+	`,
+			[article_id]
+		)
+		.then((result) => {
+			if (result.rowCount === 0) {
+				return Promise.reject({
+					status: 404,
+					msg: "Comment not found, ID doesn't exist.",
+				});
```
Comment (Rayhaan2k9) [inline models/articles.models.js:123]: This may not work when you add the test I mentioned so you will need to think of a way to ensure an empty array of comments doesn't get rejected if the article exists.

**Review by Rayhaan2k9 — CHANGES_REQUESTED** [review]
You will need to add the test and logic I mentioned. Please send another PR once complete :)

**Review by leekli — APPROVED** [review]
Well done, please feel free to merge this branch.
